nautilus_trader/adapters/dydx/websocket/client.py

```diff
@@ -105,13 +105,28 @@ def is_disconnected(self) -> bool:
     @property
     def subscriptions(self) -> set[tuple[str, str]]:
         """
-        Return the list of subscriptions.
+        Return the set of subscriptions.
+
+        Returns
+        -------
+        set[tuple[str, str]]
+            Set of subscriptions.
         """
         return self._subscriptions
 
     def has_subscription(self, item: tuple[str, str]) -> bool:
         """
         Return true if the connection is already subscribed to this topic.
+
+        Parameters
+        ----------
+        item : tuple[str, str]
+            Topic name.
+
+        Returns
+        -------
+        bool
+            Whether the client is already subscribed to this topic.
```

Comment:
I think it's best to add only additional info relative to the return type for the description, in this case it's been covered in the main description already.

Reply:
Makes sense. Thanks!